contrib/win32/openssh/OpenSSHBuildHelper.psm1

```diff
@@ -542,18 +542,22 @@ function UnInstall-OpenSSH
         [string]$OpenSSHDir = "$env:SystemDrive\OpenSSH"
     )
 
+    if (-not (Test-Path $OpenSSHDir))
```

Comment:
this should be independent of stopping the ssh-agent.. can you have this check down?